src/main/sensors/gyro.c

```diff
@@ -140,6 +140,10 @@ typedef struct gyroSensor_s {
     timeUs_t yawSpinTimeUs;
     bool yawSpinDetected;
 #endif // USE_YAW_SPIN_RECOVERY
+
+#ifdef USE_GYRO_DATA_ANALYSE
```

Comment:
The `#include sensors/gyroanalyse.h` in https://github.com/betaflight/betaflight/pull/6428/files#diff-59167518290ca93ea0a3471df4e48bbbR75 has to be made conditional to `USE_GYRO_DATA_ANALYSE` as well, to avoid breaking tests because they can't build with `arm_math.h`.

Reply:
Thanks, didn't think about doing it that way